mappings/servers/minelandnetwork/metadata.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "minelandnetwork",
+    "name": "Mineland Network",
+    "description": "Welcome to Mineland Network — the best Cracked Minecraft server on the planet!",

```

Comment:
As stated in the PR template & our docs, the description can't be longer than 50 characters (this includes spaces)